docs/release-notes/0.11/release-0.11.md

```diff
@@ -46,6 +46,20 @@ The following table describes the oldest version of the worker that the current
 
 #### Forward compatibility
 
+The following table describes the oldest version of .NET for Apache Spark release that the current version is compatible with.
```

Comment:
How about 'The following table describes the oldest version of .NET for Apache Spark release that the current Microsoft.Spark.Worker version is compatible with.'

Reply:
Changed!